cmk/base/plugins/agent_based/fortigate_ips.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+# author: Oguzhan Cicek, OpenSource Security GmbH - oguzhan(at)os-s.de
+
+#Example Input:
+#.1.3.6.1.2.1.1.1.0 testname
+#.1.3.6.1.2.1.1.2.0 .1.3.6.1.4.1.12356.101.1.15000
+#[...]
+#.1.3.6.1.4.1.12356.101.9.2.1.1.1.102 0
+#.1.3.6.1.4.1.12356.101.9.2.1.1.1.103 0
+#.1.3.6.1.4.1.12356.101.9.2.1.1.1.104 0
+#.1.3.6.1.4.1.12356.101.9.2.1.1.1.105 0
+#.1.3.6.1.4.1.12356.101.9.2.1.1.1.106 0
+#.1.3.6.1.4.1.12356.101.9.2.1.1.2.102 0
+#.1.3.6.1.4.1.12356.101.9.2.1.1.2.103 0
+#.1.3.6.1.4.1.12356.101.9.2.1.1.2.104 0
+#.1.3.6.1.4.1.12356.101.9.2.1.1.2.105 0
+#.1.3.6.1.4.1.12356.101.9.2.1.1.2.106 0
+#[...]
+
+#Example GUI Output:
+#OK     FortiGate IPS Info: 0
+#       Intrusions per second: detected: 0.00, blocked: 0.00
+#OK     FortiGate IPS Info: 1
+#       Intrusions per second: detected: 0.00, blocked: 0.00
+#OK     FortiGate IPS Info: 2
+#       Intrusions per second: detected: 0.00, blocked: 0.00
+#OK	    FortiGate IPS Info: 3
+#       Intrusions per second: detected: 0.00, blocked: 0.00
+#OK     FortiGate IPS Info: 4
+#       Intrusions per second: detected: 0.00, blocked: 0.00
+
+from .agent_based_api.v1.type_defs import (
+    DiscoveryResult,
+    CheckResult,
+    StringTable,
+)
+from .agent_based_api.v1 import (
+    register,
+    get_rate,
+    Service,
+    startswith,
+    Metric,
+    SNMPTree,
+    check_levels,
+    get_value_store,
+)
+from typing import List, Dict, Mapping, Tuple
+import time
+
+Section = Dict[str, Dict[str, int]]
+
+
+def parse_fortigate_ips(string_table: List[StringTable]) -> Section:
+    parsed = {}
+    for i in range(len(string_table[0])):
+        parsed.update({
+            str(i): {
```

Comment:
One question: Would it make sense to use the OID ending as item here, or is this information not useful?